src/ports/postgres/modules/deep_learning/test/madlib_keras.sql_in

```diff
@@ -270,6 +270,17 @@ FROM (SELECT * FROM keras_out_summary) summary;
 
 SELECT assert(model_data IS NOT NULL , 'Keras model output validation failed') FROM (SELECT * FROM keras_out) k;
 
+-- Validate metrics=NULL works fine
+DROP TABLE IF EXISTS keras_saved_out, keras_saved_out_summary;
+SELECT madlib.madlib_keras_fit(
+'cifar_10_sample_batched',
+'keras_saved_out',
+'model_arch',
+1,
+$$ optimizer=SGD(lr=0.01, decay=1e-6, nesterov=True), loss='categorical_crossentropy'$$::text,
+$$ batch_size=2, epochs=1, verbose=0 $$::text,
+3);
+
 DROP TABLE IF EXISTS cifar10_predict;
```

Comment:
maybe add a test for ` $$ optimizer=SGD(lr=0.01, decay=1e-6, nesterov=True), loss='categorical_crossentropy, metrics=[]'$$::text,`